content/docs/next-release/operator.md

```diff
@@ -674,6 +674,47 @@ spec:
 
 <2> Disables creation of default index templates.
 
+### Storage plugin
+
+Storage type `grpc-plugin` allows using Jaeger with 3rd party storage implementations.
+
+Follows an example of Jaeger CR with `allInOne` strategy and Clickhouse storage plugin.
```

Comment:
The following is an example of a Jaeger CR using the `allInOne` deployment strategy and the Clickhouse storage plugin.  Refer to [jaeger-clickhouse](https://github.com/pavolloffay/jaeger-clickhouse) for documentation and a fully functional example.

Reply:
@pavolloffay are you using @JStickler's version for this paragraph?

Reply:
now yes, I didn't push this one before